code/game/objects/structures/watercloset.dm

```diff
@@ -578,12 +572,83 @@
 	new /obj/item/stack/sheet/metal (loc, 3)
 	qdel(src)
 
-
-
 /obj/structure/sink/kitchen
 	name = "kitchen sink"
 	icon_state = "sink_alt"
 
+/obj/structure/sink/well
+	name = "well"
+	desc = "A well, used to get water from an underground reservoir."
```

Comment:
I know this might be the wrong place for this, but... d-does Lavaland have an aquifer? I thought it was just... lava.

Reply:
In the ash den theirs legit open water pond... Its kinda ment to be drawing for that